pyanaconda/modules/payloads/source/nfs/initialization.py

```diff
@@ -74,9 +79,24 @@ def run(self):
         raise SourceSetupError(
             "Nothing useful found for NFS source at {}".format(self._url))
 
-    def _mount_nfs(self):
-        options, host, path = parse_nfs_url(self._url)
+    @staticmethod
+    def _split_iso_from_path(path):
+        """Split ISO from NFS path.
+
+        NFS path could also contain pointer to ISO which should be mounted. Problem of this
+        is that NFS path with ISO cannot be mounted as NFS mount. We have to split these
+        before mount.
+
+        :param path: path on the NFS server which could point to ISO
+        :return: tuple of path, iso_file_name; is_file_name is empty if no ISO is part of the path
+        :rtype: tuple (str, str)
+        """
+        if path.endswith(".iso"):
```

Comment:
I totally agree with you but yes we depend on that:

https://github.com/rhinstaller/anaconda/blob/master/pyanaconda/payload/image.py#L69

It would be great to have better solution but not sure what would that be.